src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.CrossGen.targets

```diff
@@ -397,6 +397,12 @@ Copyright (c) .NET Foundation. All rights reserved.
       <_ReadyToRunImplementationAssemblies Include="@(_ReadyToRunImplementationAssembliesWithoutConflicts)" />
     </ItemGroup>
 
+    <ItemGroup>
+      <_ReadyToRunPgoFiles Include="@(PublishReadyToRunPgoFiles)" />
+      <_ReadyToRunPgoFiles Include="@(RuntimePackAsset)"
+                          Condition="'%(RuntimePackAsset.AssetType)' == 'pgodata' and '%(RuntimePackAsset.Extension)' == '.mibc' and ('$(PublishReadyToRunUseRuntimePackOptimizationData)' == 'true' or '$(PublishReadyToRunUseRuntimePackOptimizationData)' == '')" />
```

Comment:
Nit: `Condition` is not aligned with `Include`.  I would either use `'$(PublishReadyToRunUseRuntimePackOptimizationData)' != 'false'` or default that property to `true` if not set.